src/main/scala/no/ndla/articleapi/service/converters/HTMLCleaner.scala

```diff
@@ -29,15 +29,16 @@ trait HTMLCleaner {
       moveImagesOutOfPTags(element)
       removeComments(element)
       removeNbsp(element)
-      removeEmptyTags(element)
+      (1 to 3).foreach(_ => removeEmptyTags(element))
```

Comment:
Det er for å være sikker på at alle tomme tagger blir fjernet. Er ikke helt fornøyd med denne måten å gjøre det på, men Jsoup har sine begrensninger.
Gitt følgende html: `<p><strong></strong></p>` så skal alt bli borte.
Siden `removeEmptyTags` her henter ut listen over tags den skal fjerne, deretter fjerner den dersom den ikke har noen underelementer er den avhengig av at `strong` fjernes først, og deretter `p`. Det fungerer ikke, og den enkleste måten jeg kom på var å kalle den mere enn en gang for å være sikker.

Den aller beste måten å fikse dette her på ville være å iterere gjennom hele DOM-treet, depth-first, og fjerne elementer fra leaf-noder først, deretter gå videre til parents.
Jsoup har en måte å iterere gjennom treet på denne måten, men den støtter ikke fjerning av noder mens man itererer. Og da er vi like langt :)